app/services/platforms/twitch.ts

```diff
@@ -272,10 +272,12 @@ export class TwitchService
       this.getHasUpdateTagsPermission(),
     ]);
 
-    let tags: TTwitchTag[] = [];
-    if (hasUpdateTagsPermission) {
+    let tags: TTwitchTag[] = this.state.settings.tags;
+
+    if (hasUpdateTagsPermission && tags.length === 0) {
```

Comment:
Can you help me understand this change?  I'm a little unsure what this is doing.